internal/gapicgen/generator/git.go

```diff
@@ -134,7 +134,7 @@ func CommitsSinceHash(gitDir, hash string, inclusive bool) ([]string, error) {
 // hash for the given gitDir.
 func UpdateFilesSinceHash(gitDir, hash string) ([]string, error) {
 	out := bytes.NewBuffer(nil)
-	c := command("git", "diff-tree", "--no-commit-id", "--name-only", "-r", fmt.Sprintf("%s..HEAD", hash))
+	c := command("git", "diff-tree", "--no-commit-id", "--name-only", "--diff-filter=ACMR", "-r", fmt.Sprintf("%s..HEAD", hash))
```

Comment:
Mind adding a comment explaining `ACMR`? I found the PR description helpful. :smile:

Reply:
Sure thing, inlining something useful.